### ISSUE TITLE
Add row and column total to table visualization

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#30598]
**Is your feature request related to a problem? Please describe.**
Several customers requested to add column and row totals in normal tables

**Describe the solution you'd like**
A toggle to add row and table columns to tables

**Describe alternatives you've considered**
Calculating those values via SQL, but leaves the GUI world outside

**How important is this feature to you?**
Requested by several customers

**Additional context**
NA

**Comment by bhernandezc** [comment]
In our organization, we are interested on this new feature, specially if the total row can be fixed.

Sometimes, in our organization we need to have a quick look in the total of the table and then analyze deeply row by row.
It's not comfortable to scroll just to look at the total. 

We think it might be viable to add it as an optional configuration of the table.

**Comment by matblissim** [comment]
+1 would be useful

**Comment by SoniaBocca** [comment]
+1 - this would be really useful for us. 
Do you have any idea when it might be available?

**Comment by jonshubeli** [comment]
Yes!!! These type of features are needed to make Metabase a reasonable replacement for Power BI. Today we have to use pivot table in Metabase, but load much slower that cached standard tables. 

And I also agree that the total row should be fixed. 

Voting and hoping for this one.

**Comment by matblissim** [comment]
pretty annoying to not have this basic feature
